tcf_website/static/reviews/new_review.js

```diff
@@ -83,18 +86,19 @@ jQuery(function($) {
             $.each(data.results, function(i, instr) {
                 $("<option />", {
                     val: instr.id,
-                    text: instr.first_name + " " + instr.last_name
+                    text: instr.last_name + ", " + instr.first_name
```

Comment:
I agree last name first makes searching easier, but are we concerned at all that this is the only place on the site with this format (as far as I can tell)?

Reply:
This is how it was on the old site so I think it should be alright

Plus the new instructor dropdown also has the astname-firstname format